### ISSUE TITLE
fugitive#statusline augroup / vim-flagship

### ISSUE THREAD

**Issue by alexanderjeurissen** [tpope/vim-fugitive#780]
I'm looking into simplifying my statusline and thought vim-flagship would be a good fit. However I don't like the default fugitive statusline that is added at:
https://github.com/tpope/vim-fugitive/blob/master/plugin/fugitive.vim#L3050

I'd rather use a custom flag like this:
```vim
    autocmd User Flags call Hoist('buffer', 'fugitive#head(7)')
```
I'm having trouble disabling the flag that is added in augroup fugitive_statusline because disabling the autocmd doesn't prevent Hoist from getting called.

I tried the following approaches to disable the autocmd group:
```vim
augroup! fugitive_statusline
```

```vim
augroup fugitive_statusline
  autocmd!
augroup end
```

```vim
autocmd! User Flags
```

All those methods don't have the desired effect however because executing
```vim
echo flagship#flags_for("buffer")
```
results in: 
```
['%##%{flagship#call(''fugitive#statusline'')}%*']
```

Maybe I'm missing something rather obvious and if so please do tell.

**Comment by tpope** [comment]
You can set `g:flagship_skip` to a regex matching functions to skip. This isn't documented but probably should be.

**Comment by lgalke** [comment]
I ran into the same issue and solved it by creating `after/ftplugin/fugitive.vim` and cleared the `augroup fugitive_statusline` there. This solution also works in case you want to add fugitive#statusline with another priority or highlighting group.
I noticed that using vim-signify or vim-gitgutter essentially disables correct highlighting group for the fugitive#statusline added by fugitive itself. Thus I had to add it explicitly with another highlighting group but the position argument was still ignored.
